systems/controllers/PlanEval.m

```diff
@@ -1,4 +1,4 @@
-classdef PlanEval
+classdef PlanEval < handle
```

Comment:
Previously, PlanEval was not a `handle`, but it carried on around as a `SharedDataHandle`. That made its pass-by semantics unnecessarily complicated (it had neither the benefits of a handle nor of a pass-by-value class). Making it a handle lets us delete `PlanEvalData.m` and clean up the whole system a bit.